apps/ejabberd/src/mongoose_api_users.erl

```diff
@@ -98,7 +98,7 @@ maybe_register_user(Username, Host, Password) ->
             ?ERROR;
         {error, exists} ->
             maybe_change_password(Username, Host, Password);
-        ok ->
+        _ ->
```

Comment:
Right, I'm all for `{ok, Acc}` when the handler does what it should without problems. Also I'm not sure if this is subject of this PR or maybe some other.